src/components/Chat/Chat.tsx

```diff
@@ -323,3 +347,22 @@ class PickerMenuInner extends React.Component<{
 }
 
 const PickerMenu = onClickOutside(PickerMenuInner);
+
+class ReactionMenuInner extends React.Component<{
+  addEmoji: (emoji: EmojiData) => void;
+  closeMenu: Function;
+}> {
+  handleClickOutside = () => {
+    this.props.closeMenu();
+  };
+  render() {
+    return (
+      <div style={{ position: 'absolute', bottom: '60px' }}>
+        <div style={{ fontSize: '24px' }}>
+          🤣😄😢❤️😮👍
```

Comment:
Were you planning to only use these emojis as reactions instead of the big emoji picker menu? Also would you mind if I tried to finish this PR?

Reply:
Yeah, I tried it originally but it seems excessive and unwieldy. And go for it if you like, i might have some ideas on how reacts are stored in messages